ADAL/tests/ADAcquireTokenTests.m

```diff
@@ -170,7 +170,7 @@ - (void)testInvalidBrokerRedirectURI
     [context setCredentialsType:AD_CREDENTIALS_AUTO];
     [context acquireTokenWithResource:TEST_RESOURCE
                              clientId:TEST_CLIENT_ID
-                          redirectUri:[NSURL URLWithString:@"urn:ietf:wg:oauth:2.0:oob"]
+                          redirectUri:[NSURL URLWithString:@"invalid://redirect_uri"]
```

Comment:
This change should not be necessary. In non-broker ADAL builds `@"urn:ietf:wg:oauth:2.0:oob"` MUST still be seen an invalid redirect URI.